src/commands/tenmans.ts

```diff
@@ -134,7 +136,63 @@ class TenmansCloseSubcommand extends MessageExecutable<CommandInteraction> {
 
     // Teardown - clear current queue
     tenmansQueue = []
-    await activeTenmansMessage.delete();
+    await activeTenmansMessage?.delete();
+  }
+}
+
+class TenmansVoteSubcommand extends RegisteredUserExecutable<CommandInteraction> {
+  async afterUserExecute() {
+    if (activeTenmansMessage) {
+      this.interaction.reply({
+        content: "You should have been looking more closely. There's already a queue - use that one instead!",
+        ephemeral: true
+      });
+
+      return;
+    }
+
+    const queueChannel = botConfig.queueMsgChannel as TextChannel;
+    if (!queueChannel) {
+      this.interaction.reply({
+        content:
+          "No queue message channel configured. Please set channel id using '/config defaultChannel'.",
+        ephemeral: true,
+      });
+
+      return;
+    }
+
+    if (!tenmansQueue?.length) {
```

Comment:
If you're checking for length being equal to zero, I'd recommend being explicit here

Reply:
This statement also handles uninitialized values like null, not just length being zero. The optional chaining feature of JS makes it a bit more concise to check the null and empty list case.

Reply:
I think we just have to accept it being harder to read then if you want to keep it concise.